packages/ove-app-videos/README.md

```diff
@@ -0,0 +1,37 @@
+# Videos App
+
+This app supports playing videos using the OVE framework. There are two players available, `HTML5`, which is a generic player for any video, and `YouTube` which is based on the [YouTube IFrame Player API](https://developers.google.com/youtube/iframe_api_reference).
```

Comment:
"It provides a generic player for any video (`HTML5`), and a player based on the [YouTube IFrame Player API](https://developers.google.com/youtube/iframe_api_reference) (`YouTube`)."